### PR TITLE
chart: enable estimator component for scheduler

### PR DESCRIPTION
**Pull request by calvin0327**
Signed-off-by: calvin <wen.chen@daocloud.io>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
currently, we cannot enable estimator for schedler. we need a switch to open the feature in weight-schedule.

image

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
please correct me if not.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by calvin0327** [comment]
@Poor12 PTAL

File: charts/karmada/values.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ installMode: "host"
 clusterDomain: "cluster.local"
 
 ## @param components component list
-components: []
+components:
```
Comment (Poor12) [inline charts/karmada/values.yaml:24]: I wonder if these format changes automatically discovered by the script or how are they identified.

Comment (calvin0327) [reply charts/karmada/values.yaml:24]: The change is caused by vscode plugin `vscode-helm`.  It is an automatic formatting tool

**Comment by calvin0327** [comment]
@Poor12 any suggests?

**Comment by Poor12** [comment]
Please update the README.md together.

File: charts/karmada/README.md [hunk 2]
```diff
@@ -121,7 +121,15 @@ Execute command (switch to the `root` directory of the repo, and sets the `curre
 kubectl config use-context member
 helm install karmada-agent -n karmada-system --create-namespace ./charts/karmada
 ```
+
 ### 2. Install component
+Before installing estimator, we should set flags of scheduler to take effect:
```
Comment (Poor12) [inline charts/karmada/README.md:126]: Hi @calvin0327, I guess you misunderstood what I mean. We do not need to add specific words for a paramter, just add it in https://github.com/karmada-io/karmada/tree/master/charts/karmada#common-parameters is ok.

Comment (calvin0327) [reply charts/karmada/README.md]: Sorry. updated

**Comment by RainbowMango** [comment]
@calvin0327 The /retest command not working. I just re-triggered the tests.

File: charts/karmada/templates/karmada-scheduler.yaml [hunk 3]
```diff
@@ -50,6 +50,9 @@ spec:
             - --kubeconfig=/etc/kubeconfig
             - --bind-address=0.0.0.0
             - --secure-port=10351
+            - --feature-gates=Failover=true
```
Comment (Poor12) [inline charts/karmada/templates/karmada-scheduler.yaml:53]: Failover has changed to Beta now and is enabled by default. So we do not need it anymore.

Comment (calvin0327) [reply charts/karmada/templates/karmada-scheduler.yaml:53]: the pr seem is not needed, so close it.

**Pull request closed without merging**